### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by r-ash**
This PR will

* Add translations for progress bars
* Also translate some errors from the model

Rich please could you check the translations and change as you see fit? They are just coming from google translate and are probably nonsense in many cases.

As mentioned in hintr PR https://github.com/mrc-ide/hintr/pull/155 this doesn't fix the translations of options for sex or age (one option is "all ages"). 

For these we have a mechanism to translate in that the options can have an id (which would stay as English version) and a label which we could have translated. At the moment though for sex both of these are coming from the data and we assume label is the same as the ID. And indeed the data as users download it will have male, female, both in it. So the question is - do we want to translate so what they see in the app is in French then when they download the data it won't be translated which may be confusing (I think that is what will happen with calendar quarter atm - where the download will show CY2019Q3 but the app will show August 2019 or similar).

Any thoughts on that? I'm tempted to say we do the translation so it is consistent with calendar quarter.

### CODE REVIEW

File: inst/traduire/en-translation.json [hunk 1]
```diff
@@ -97,5 +97,16 @@
     "INDICATOR_LABEL_ANC_PREVALENCE": "HIV prevalence among ANC attendees",
     "INDICATOR_LABEL_ANC_ART_COVERAGE": "ART coverage among ANC attendees prior to first ANC",
     "OPTIONS_ART_TIME_VARYING_ART_ATTEND_LABEL": "Time-varying neighbouring district ART attendance",
```
[the remark this answers is not in the document]
Comment (r-ash) [reply inst/traduire/en-translation.json]: Could you add translations for those? I would just be putting them through google translate.

File: R/run-model.R [hunk 2]
```diff
@@ -231,47 +231,45 @@ Progress <- R6::R6Class("Progress", list(
...
+    index <- self$find_step(step_name)
     self$progress[[index]]$complete <- TRUE
   },
-  find_step = function(message) {
-    steps <- vapply(self$progress, function(step) {
-      step$name == message
-    }, logical(1))
+  find_step = function(step_name) {
+    steps <- names(self$progress) == step_name
     if (sum(steps) != 1) {
```
[the remark this answers is not in the document]
Comment (r-ash) [reply R/run-model.R]: Yep it shouldn't ever really happen unless some coding mistake. I've updated the way that class is working.

File: R/model.R [hunk 3]
```diff
@@ -983,10 +988,11 @@ artnum_mf <- function(calendar_quarter, art_number, naomi_mf) {
       dplyr::filter(out_quarter_id > min_data_quarter - 4L,
                     out_quarter_id <= max_data_quarter)
 
-    if(nrow(dat) == 0)
-      stop(paste0("No ART data found for quarter ", calendar_quarter, ".\n", 
-            "Set calendar_quarter = NULL if you intend to include no ART data."))
-    
+    if(nrow(dat) == 0) {
```
[the remark this answers is not in the document]
Comment (r-ash) [reply R/model.R]: Thanks, done

File: R/model.R [hunk 4]
```diff
@@ -893,10 +896,11 @@ anc_testing_artcov_mf <- function(year, anc_testing, naomi_mf) {
     )
   } else {
 
-    if(!all(year %in% anc_testing$year))
-      stop(paste("ANC testing data not found for year",
-                 setdiff(year, anc_testing$year)))
-    
+    if(!all(year %in% anc_testing$year)) {
+      stop(t_("ANC_DATA_MISSING_FOR_YEAR",
```
[the remark this answers is not in the document]
Comment (r-ash) [reply R/model.R]: Fixed this and the other instances mentioned. Could you check the pluralisation of the messages please? I have mostly just stuck an s somewhere that looks appropriate.

File: inst/traduire/en-translation.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (richfitz) [reply inst/traduire/en-translation.json]: done

File: R/model.R (hunk 4 above)
[the remark this answers is not in the document]
Comment (richfitz) [reply R/model.R]: done, thanks - you were close! (see d12c83d)

**Pull request merged**